src/cljs/nr/gameboard/player-stats.cljs

```diff
@@ -43,19 +43,22 @@
          (ctrl :memory [:div label (when (neg? unused) [:div.warning "!"])]))))))
 
 (defn- display-special-memory
-  [memory]
+  ([memory] (display-special-memory memory false))
+  ([memory icon?]
   (when-let [only-for (->> (:only-for memory)
                            (filter #(pos? (:available (second %))))
                            (into {})
                            not-empty)]
     [:div
-     (str "("
+     [:<> "("
           (join "), (" (for [[mu-type {:keys [available used]}] only-for
-                             :let [unused (max 0 (- available used))]]
-                         (str unused " of " available
-                              " " (capitalize (name mu-type))
-                              " [mu] unused")))
-          ")")]))
+                             :let [
+                               unused (max 0 (- available used))
+                               mu-type-name (capitalize (name mu-type))
+                              ]]
+                         (if icon? [:<> unused "/" available " " mu-type-name " " [:span.anr-icon.mu]]
+                             (tr [:game.special-mu-count] unused available mu-type-name))))
+          ")"]])))
```

Comment:
Looks like since we are needing to use span elements to display the icon, we can't use any string functions like join like we used to. Instead we'll have to loop through the mu-types and generate an element for each one. I messed around with it for a bit and managed to get it to display using the code below.
![image](https://user-images.githubusercontent.com/5953664/133892895-6d8d7cae-b580-45e5-8418-80c300d1f3e0.png)

```suggestion
(defn- display-special-memory
  ([memory] (display-special-memory memory false))
  ([memory icon?]
   (when-let [only-for (->> (:only-for memory)
                            (filter #(pos? (:available (second %))))
                            (into {})
                            not-empty)]
     [:<>
      (doall
       (for [[mu-type {:keys [available used]}] only-for
             :let [unused (max 0 (- available used))
                   mu-type-name (capitalize (name mu-type))]]
         ^{:key mu-type-name}
         [:div (if icon? [:<> unused "/" available " " mu-type-name " " [:span.anr-icon.mu]]
                   (tr [:game.special-mu-count] unused available mu-type-name))]))])))
```

Reply:
Thank you so much @cardboardtech !

Reply:
happy to help